delegator/adder/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "adder"
-version = "4.1.0"
+version = "4.2.0"
```

Comment:
I'm assuming you mean `4.2` for `ink` and `ink_e2e` `[dependencies]` version, correct?
since the ` 0.0.0 ` is the required schema for `version` number under the `[package]` section inside the Cargo.toml file.

Thanks for the feedback, I will proceed with this request today evening

Reply:
done